change/@internal-react-composites-97ee5a6b-91ac-441d-860b-6e757a0d3cc3.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "Make file sharing responsive by adding style changes",
```

Comment:
We need to adjust z-index, otherwise messages lay on top of participant pane
<img width="200" alt="Screen Shot 2022-05-31 at 12 35 22 PM" src="https://user-images.githubusercontent.com/97124699/171270752-a797692d-a14f-4e13-81df-2c46a70b1931.png">
.

Reply:
Done